src/app/options/options.component.ts

```diff
@@ -100,7 +100,37 @@ export class OptionsComponent implements OnInit {
     const modalRef = this.modalService.open(ModalComponent);
     modalRef.componentInstance.title = 'About this project';
     modalRef.componentInstance.content = `
-        <strong>Hi</strong> there!
+        <p>
+            Thank you for giving the official Kubernetes Release Notes website
+            a try! The project aims to provide you with the best possible user
+            experience when it comes to finding out what changed in Kubernetes.
+        </p>
+        <p>
+            We would love to get your feedback about this website, especially
+            in terms of usability, look and feel and how to improve it even
+            further regarding your personal needs. In order to achive this goal
+            we prepared a survey and we would appreciate if you could take your
+            time to fill it out and help us to improve.
```

Comment:
This paragraph is grammatically fine but a bit awkward at times. I would suggest the following:

```
 We would love feedback on your experience using this website. We are particularly interested in:

 - how easy it is to use and find what you are looking for

 - the look and feel 

 - how to improve it further for your specific use case

We would appreciate it if you could take a few minutes to fill out our survey and help us to improve!
```

Reply:
Okay, I changed it regarding your suggestion.